src/main/include/Robot.h

```diff
@@ -35,4 +35,5 @@ class Robot : public frc::TimedRobot {
    //subject to adjustment
   void joy_val_to_mps(double& val) { val *= 4; }
```

Comment:
There's no real reason; I initially wrote those functions when I was working with a programmer (outside of robotics) who loved modifying values like this and it rubbed off on me. This implementation is messy overall, I'll change it to re-assign the value and I'll put the definition in Robot.cpp (especially since we may modify or complicate the function to better match the speed profile of the old robot).